packages/lodestar/src/chain/forkChoice/interface.ts

```diff
@@ -12,7 +12,7 @@ export interface ILMDGHOST {
   stop(): Promise<void>;
   addBlock(info: BlockChainInfo): void;
   addAttestation(blockRootBuf: Uint8Array, attester: ValidatorIndex, weight: Gwei): void;
-  head(): Uint8Array;
+  head(): {slot: Slot; root: Uint8Array};
```

Comment:
Can we break this out into `headBlockRoot` and `headSlot`?

When we convert to protoarray, we can take time to expose something with all head data like a `head(): BlockChainInfo`

Reply:
What would we get other than huge bulk of changes and test fixes?
Feel like this is a good start to add more info when converting to protoarray. We already have those data in forkchoice but we aren't returning it

Reply:
or did you mean to rename slot, root into headBlockRoot and headBlockSlot?

Reply:
Sorry, I guess it doesn't have much to do with protoarray.

I was thinking we can/should return _all_ the data we have, the whole `BlockChainInfo` (or similar, with Uint8Array instead of hex string). All head-related methods then become "convenience" methods that call `head`.
```
head(): BlockChainInfo {
  ...
  return this.justified.bestNode.target;
}
headStateRoot(): Uint8Array {
  return this.head().stateRoot;
}
headSlot(): Slot {
  return this.head().slot;
}
...
```